### PR TITLE
Woa2018 pt

### PR DESCRIPTION
**Pull request by Jipingnersc**
Add the python chain of to convert in-situ temperature to potential temperature for woa2018/woa2013

### CODE REVIEW

**Comment by caglartac** [comment]
Jiping, you asked for merging to master branch. You should set this to develop branch.

File: TP2a0.10/expt_01.0/blkdat.input [hunk 1]
```diff
@@ -205,17 +205,19 @@ S-Z(15-11): dp00/f/x/i=3m/1.125/12m/1m; ds=1m/1.125/4m; src_2.2.12;
   -1	  'lwflag' = longwave (SST) flag (0=none,1=clim,2=atmos)
    0	  'sstflg' = SST relaxation flag (0=none,1=clim,2=atmos,3=observed)
    0	  'icmflg' = ice mask       flag (0=none,1=clim,2=atmos,3=obs/coupled)
-1.e5      'prsbas' = msl pressure is input field + prsbas (Pa)
```
Comment (lbertino) [inline TP2a0.10/expt_01.0/blkdat.input:208]: I know that 1 Pa is not much but I think the real value is 0 when MSLP is in absolute value.

File: TP0a1.00/expt_01.0/blkdat.input [hunk 2]
```diff
@@ -187,17 +187,19 @@ S-Z(15-11): dp00/f/x/i=3m/1.125/12m/1m; ds=1m/1.125/4m; src_2.2.12;
   -1	  'lwflag' = longwave (SST) flag (0=none,1=clim,2=atmos)
    0	  'sstflg' = SST relaxation flag (0=none,1=clim,2=atmos,3=observed)
    0	  'icmflg' = ice mask       flag (0=none,1=clim,2=atmos,3=obs/coupled)
-1.e5      'prsbas' = msl pressure is input field + prsbas (Pa)
+   1      'prsbas' = msl pressure is input field + prsbas (Pa)
```
Comment (lilleannette) [inline TP0a1.00/expt_01.0/blkdat.input:190]: I think prsbas should be zero.

File: TP0a1.00/expt_01.0/blkdat.input [hunk 3]
```diff
@@ -187,17 +187,19 @@ S-Z(15-11): dp00/f/x/i=3m/1.125/12m/1m; ds=1m/1.125/4m; src_2.2.12;
...
    0	  'trcrlx' = activate lat. bound. tracer nudging  (0=F,1=T)
    1	  'priver' = rivers as a precipitation bogas      (0=F,1=T)
+   0      'triver' = total river including Greenland (0=F,1=T)
    0     'epmass' = treat evap-precip as a mass exchange (0=F,1=T)
    0     'nmrsti' = 'restart'  Restart filename. Dot and date appended. Empty: use old hycom name.
    0     'nmrsto' = 'restart'  Restart filename. Dot and date appended. Empty: use old hycom name.
    0     'nmarcs' = '' Surface archive filename. Dot and date appended. Empty: use old hycom name.
    0     'nmarcv' = '' Full    archive filename. Dot and date appended. Empty: use old hycom name.
    0     'nmarcm' = '' Mean    archive filename. Dot and date appended. Empty: use old hycom name.
+   6     'nrdflg' = (0,1,...,7) options for the radiation heat fluxes.
```
Comment (lilleannette) [inline TP0a1.00/expt_01.0/blkdat.input:205]: I thought we established that nrdflg=6 does not give good results.

File: TP0a1.00/expt_01.0/srjob.sh [hunk 4]
```diff
@@ -49,7 +50,7 @@ INITFLG="--init"
 echo "Start time in pbsjob.sh: $START"
 echo "End   time in pbsjob.sh: $END"
 # Generate atmospheric forcing :
-atmo_synoptic.sh erai $START $END 
+atmo_synoptic.sh erai+all $START $END
```
Comment (lilleannette) [inline TP0a1.00/expt_01.0/srjob.sh:53]: Should it be erai-lw?

**Pull request closed without merging**